tests/integration/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""
+Integration tests setup configurations and fixtures
+"""
+import time
+from pathlib import Path
+import pytest
+from commonlib.io_utils import get_k8s_yaml_objects
+
+
+DEPLOY_YML = "../../deploy/cloudbeat-pytest.yml"
+
+
+@pytest.fixture(scope='module', name='start_stop')
+def fixture_start_stop(k8s, api_client, cloudbeat_agent):
+    """
+    This fixture starts cloudbeat/agent on test module setup and
+    stops on teardown of the test module
+    @param k8s: Kubernetes client object
+    @param api_client: Docker api / FileSystem api
+    @param cloudbeat_agent: Cloudbeat configuration
+    @return: Kubernetes object, Api client, Cloudbeat config
+    """
+    file_path = Path(__file__).parent / DEPLOY_YML
+    if k8s.get_agent_pod_instances(agent_name=cloudbeat_agent.name,
+                                   namespace=cloudbeat_agent.namespace):
+        k8s.delete_from_yaml(get_k8s_yaml_objects(file_path=file_path))
+    k8s.start_agent(yaml_file=file_path, namespace=cloudbeat_agent.namespace)
+    time.sleep(5)
+    yield k8s, api_client, cloudbeat_agent
+    k8s_yaml_list = get_k8s_yaml_objects(file_path=file_path)
+    k8s.delete_from_yaml(yaml_objects_list=k8s_yaml_list)  # stop agent
+
+
+@pytest.fixture(name='fixture_data')
+def fixture_data(start_stop, k8s, cloudbeat_agent):
```

Comment:
why use `start_stop` as a fixture here, instead of just calling the function?

Reply:
Because start_stop fixture allows to start cloudbeat before test actions and stop it after test execution, while function call requires additional logic implementation.
Fixture_data called several times during the test run, but start_stop only once because of scope definition.
More about fixtures may be found [here](https://docs.pytest.org/en/6.2.x/fixture.html)